Tweaks and small improvements to the handbook (#153)
canonical/ubuntu-maintainers-handbook: Concepts.md

#### Restricted

> Packages supported by the Ubuntu team, which are not available under a

Reviewer: ```suggestion
Packages provided by the Ubuntu community, which are not available under a
```

AND/OR do not try to explain them and just refer to https://help.ubuntu.com/community/Repositories - WDYT?

Reviewer: supported is a very dangerous word, very very much :-) Misunderstanding pre-programmed

Also there is no "Ubuntu team" hence suggesting community if you want to keep it instead of referring

> completely free license.

Reviewer: I'm sure a lawyer will hunt you for what "completely free" is vs "free"

those in a different suite if overlaid on top of it. Some examples of partial
suites are `backports`, `proposed`, `security`, and `updates`.

#### updates

Once an Ubuntu version is released, the `series` pocket stops receiving new
package versions, but updates to the initial set are still uploaded with
diverse fixes. Versions with regular bug fixes, or potential new features,
are present in `series-updates`.

#### security

The `series-security` pocket carries updates to packages in `series` which
bring security fixes. This suite is maintained by the security team.

#### proposed

When any package is uploaded to Ubuntu, it will land in `series-proposed`. This

> suite is used for testing the package to make sure it buils and that tests pass

Reviewer 2: ```suggestion
suite is used for testing the package to make sure it builds and that tests pass
```

before the uploaded version can be copied into the release itself. During the

> development of a new release, packages which are considered in good state

Reviewer 2: ```suggestion
development of a new release, packages that are considered in a good state
```

migrate from `series-proposed` to `series`. After the release, updates to the
packages migrate from `series-proposed` to `series-updates` (as part of the
SRU process).

#### backports

Software in `series-backports` may not have been tested as extensively as that
contained in the above examples, although it includes newer versions of some

> applications which may provide useful features. Those versions are usually

Reviewer 2: ```suggestion
applications that may provide useful features. Those versions are usually
```
